### ISSUE TITLE
Test Subjects Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4899]
# Page to Test:

Subjects - https://openlibrary.org/subjects/

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by ma8642** [comment]
@bpmcneilly I will work on this!

**Comment by ma8642** [comment]
@bpmcneilly I have filed issue tickets for all the bugs I found on this page.

**Comment by RayBB** [comment]
This one actually was done by @ma8642 as seen above but nobody ever got around to closing it.
